R/load_precomputed_layouts.R

```diff
@@ -60,7 +67,7 @@ ReadMPX_layouts <- function (
   # Load hive-styled parquet files
   coords <- arrow::open_dataset(temp_layout_dir)
   coords <- coords %>%
-    select(any_of(c("x", "y", "z", "sample")), component, graph_projection, layout) %>%
+    select(any_of(c("name", "x", "y", "z", "sample")), component, graph_projection, layout) %>%
```

Comment:
Here we keep the name column, but in line 324 in 'load_cell_graphs' we remove it from the cellgraph@layout elements. I am not sure if relevant, but that would lead to a discrepancy between using taht function or this one directly, right?

Reply:
That's correct. `ReadMPX_layouts` returns a list of lists of tibbles where each tibble contains an additional column with node names. The reason why its a list of list is because each component can have multiple pre-computed layouts. The order of the components in that list will probably not match the order of the components in a Seurat object generated from the same PXL file. The layout files are not ordered in any specific way in the PXL file, so they will be loaded in the order they appear.

I think this discrepancy is fine since the way layouts are stored in the Seurat object is quite different anyways.

Reply:
Okay, that's fine with me then!